dev/RadioButtons/RadioButtons.cpp

```diff
@@ -67,40 +67,37 @@ void RadioButtons::OnGettingFocus(const winrt::IInspectable&, const winrt::Getti
     if (auto const repeater = m_repeater.get())
     {
         auto const inputDevice = args.InputDevice();
-        if (inputDevice == winrt::FocusInputDeviceKind::Keyboard || inputDevice == winrt::FocusInputDeviceKind::GameController)
+        if (inputDevice == winrt::FocusInputDeviceKind::Keyboard)
```

Comment:
Gamepad won't do anything when focus changes?

Reply:
Correct, selection follows focus should not occur for gamepad since it would be impossible to escape the radio buttons control without selecting the last (or first) radio button.

Reply:
Isn't this from the other PR?